src/Directory.Build.props

```diff
@@ -69,6 +69,9 @@ If you want to see the [full set of changes made in Akka.NET v1.5.14, click here
 | 1       | 227  | 25   | Drew                |
 | 1       | 1    | 1    | szaliszali          |</PackageReleaseNotes>
   </PropertyGroup>
+  <ItemGroup Label="Analyzers">
+    <PackageReference Include="Akka.Analyzers" Version="0.1.2" PrivateAssets="all" />
```

Comment:
Included this package as a private asset in `Directory.Build.props` to the analysis would be applied solution-wide.